### PR TITLE
Added Python Crash Course

### PR DESCRIPTION
**Pull request by CyberSecByte**
## What does this PR do?
Add resource

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by CyberSecByte** [comment]
Attention here please @eshellman @shrn01

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1002,6 +1002,7 @@
 * [Python Built in Functions A to Z Tutorial and Examples](https://www.youtube.com/playlist?list=PLrJGwAG1U62RW_hGGnk7xXG0LciHkT54X) - Brainy Things
 * [Python Course](https://www.python-course.eu)
 * [Python Course from scratch](https://scrimba.com/playlist/pNpZMAB) - Olaf Paulson (scrimba)
+* [Python Crash Course](https://www.youtube.com/watch?v=f79MRyMsjrQ) - Programming With Mosh (Youtube)
```
Comment (eshellman) [inline courses/free-courses-en.md:1005]: Don't invent titles! Use the title given by the resource.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -988,6 +988,7 @@
 * [Learn Python](https://pythonspot.com)
 * [Learn Python - Free Interactive Python Tutorial](https://www.learnpython.org)
 * [Learn Python From Scratch](https://www.udemy.com/learn-python-from-scratch-w/) - MD. Hasanur Rahaman Hasib (Udemy)
+* [Learn Python Programming - Python Course](https://www.youtube.com/watch?v=f79MRyMsjrQ) - Programming With Mosh (Youtube)
```
Comment (eshellman) [inline courses/free-courses-en.md:991]:
```suggestion
* [Learn Python Programming - Python Course](https://www.youtube.com/watch?v=f79MRyMsjrQ) - Programming With Mosh
```

Comment (CyberSecByte) [reply courses/free-courses-en.md]: @eshellman please check and merge jt

**Review by shrn01 — APPROVED** [review]
Please fix the linting error, otherwise the PR looks good

**Comment by CyberSecByte** [comment]
> Changes look good, but looks like the URL check is timing out

Yeah idk why it's happening,  also the process ran for too long

**Comment by CyberSecByte** [comment]
Hopefully this will fix the url check

**Comment by CyberSecByte** [comment]
ok it did @shrn01 @eshellman now you can merge it

**Pull request merged**